packages/@n8n/api-types/src/dto/folders/list-folder-query.dto.ts

```diff
@@ -0,0 +1,181 @@
+import { ApplicationError, jsonParse } from 'n8n-workflow';
```

Comment:
How necessary is using `jsonParse` here, as we're using it without the extra features. Now using it brings extra production dependency.

Reply:
I get a warning when trying to use the native `JSON.parse`. We are already using the `n8n-workflow` package in the FE and the CLI (backend entry point), so it does not matter since this package is used in those two packages.